### DISCUSSION TITLE
GPX Datei hochladen - mobile Version

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#120] [Ideas]
Die Werkzeuge „hochladen (GPX,…“ und „herunterladen (GPX,…“ werden in der mobilen Version unter „Werkzeuge  ->  Zeichnen(Redlining)“ nicht angezeigt.
Was das Thema Dateien hochladen in der mobilen Version unserer Meinung nach aber wesentlich vereinfachen würde.

**Desktop:**
![image](https://github.com/gis-eni/webgis-forum/assets/124671123/9f4c1843-5c33-49c9-93aa-d41a60d11585)

**Mobil:**
![image](https://github.com/gis-eni/webgis-forum/assets/124671123/f4a8bbf8-b504-4dbe-93c2-42b9fd76e64f)

Die derzeitige alternative Vorgehensweise (mobil) über das Werkzeug „Linie“ im „Redlining“ ist für den „einfachen Anwender“ nicht wirklich intuitiv u viel zu umständlich.
Beispiel:  Ich möchte eine Wanderroute im GPX Format auf mein mobiles Endgerät laden 
Gehe ich dennoch diesen Weg, wird es bei kleineren Smartphone Formaten schwierig, da das Kontext Menü hinter 
dem „Redlining Menü“ versteckt ist und man ohne das Menü gar nicht dazu kommt die Datei hochzuladen.
Hat man es dann doch gefunden sind mehrere Schritte mit etwas Geduld durchzuführen um ans gewünschte Ziel zu gelangen

Siehe Video
Film zeigt Beispiel über dev Tools am iPhone SE

https://github.com/gis-eni/webgis-forum/assets/124671123/558a8e7a-502a-4a2e-b7f9-6490936bda1c

**Comment by gis-eni** [comment]
Mach Sinn. Die Werkzeuge kommen im nächsten Build einmal ins die mobile Variante:

<img width="282" alt="image" src="https://github.com/gis-eni/webgis-forum/assets/62870568/27608b8b-da6f-4a06-b4a4-7ea9ab34f911">

Ob das dann praktikabel genutzt werden kann, müsst bitte ihr testen.
